warehouse/templates/pages/help.html

```diff
@@ -1031,6 +1031,7 @@ <h3 id="pronunciation">{{ pronunciation() }}</h3>
       <h3>{% trans %}Resources{% endtrans %}</h3>
       <p>{% trans %}Looking for something else? Perhaps these links will help:{% endtrans %}</p>
       <ul>
+        <li><a href="{{ request.user_docs_url('/') }}">{% trans %}PyPI User Documentation{% endtrans %}</a></li>
```

Comment:
This is technically an External link, and should be highlighted as such. Might as well also add the target/rel properties.

Reply:
Fixed, thanks!